### PR TITLE
Fix Translation II

### PR DESCRIPTION
**Pull request by Carreau**
# Description

Fix more translation and put .format() outside trans._().
renable pyupgrade f-strings formatting
Use semgrep to check no incorrect syntax is used in trans

See #2453 for discussion

Will likely need to be rebased on top of #2453  for tests to pass.

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by goanpeca — APPROVED** [review]
LGTM!

**Comment by Carreau** [comment]
Rebase on master, and marked as ready.

File: .github/workflows/test_pull_requests.yml [hunk 1]
```diff
@@ -43,6 +43,25 @@ jobs:
...
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+      - name: Check Manifest
+        run: |
+          pip install --upgrade pip semgrep
+          # f"..." and f'...' are the same for semgrep
+          semgrep --error --lang python --pattern 'trans._(f"...")' napari
+          semgrep --error --lang python --pattern 'trans._($X.format(...))' napari
```
Comment (goanpeca) [inline .github/workflows/test_pull_requests.yml:64]: other patterns that could be used eventually are

trans._p(
trans._np(
trans._n(

**Review by goanpeca — APPROVED** [review]
Thanks @Carreau

**Review by sofroniewn — APPROVED** [review]
All this looks good to me. I'm fine with the additional `Check l18n syntax` CI pass, but maybe @tlambert03 would like to weigh in before merge. Thanks for checking over the strings too @goanpeca

File: .github/workflows/test_pull_requests.yml [hunk 2]
```diff
@@ -43,6 +43,25 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+      - name: Check Manifest
```
Comment (tlambert03) [inline .github/workflows/test_pull_requests.yml]:
```suggestion
      - name: Check localization formatting
```

**Review by tlambert03 — APPROVED** [review]

**Pull request merged**